src/server/pachyderm_test.go

```diff
@@ -8202,7 +8198,7 @@ func TestCancelManyJobs(t *testing.T) {
 	var err error
 	for i := 0; i < 10; i++ {
 		commits[i], err = c.StartCommit(repo, "master")
-		require.NoError(t, c.PutFile(commits[i], "file", strings.NewReader("foo"), client.WithAppendPutFile()))
+		require.NoError(t, c.PutFile(commits[i], "file", strings.NewReader("foo")))
```

Comment:
ah should have beat you to a github comment. Three of these tests (TestCancelJob, TestCancelManyJobs, TestSquashCommitRunsJob) are actually fixed by this change: https://github.com/pachyderm/pachyderm/pull/6232/files#diff-a84d233b0c21b3063c531e93ebb48872b3bb468f62b59476d459d5540fc6d7b3R298

Canceled Jobs' goroutines weren't being cancelled which would make the system hang


I just edited this line because of a discussion i had with Bryce where we can clean up some instances of "append" which could be addressed with normal PUT operations. I'm also happy to remove the change from this line

Reply:
Ah okay, then I'm fine with it either way